docs/nomenclature.md

```diff
@@ -138,6 +138,30 @@ Like the previous section, we use the `E` for the error parameter type.
 | `F[A] => ((A,Int) => B) => F[B]` | `mapWithIndex` |
 | `F[A] => ((A,Int) => G[B]) => G[F[B]]` | `traverseWithIndex` | `F: Monad`
 
+### SemigroupK
+| Type         | Method Name  | Constraints |
+|------------|--------------|-----------|
+| `F[A] => F[A] => F[A]`| `combineK` | 
+| `F[A] => Int => F[A]` | `combineNK`
+| `F[A] => F[B] => F[Either[A, B]]` | `sum` | `F: Functor`
+| `IterableOnce[F[A]] => Option[F[A]]` | `combineAllOptionK`
+
+### MonoidK
+| Type         | Method Name  | Constraints |
+|------------|--------------|-----------|
+| `F[A]` | `empty`
+| `F[A] => Boolean` | `isEmpty`
+| `IterableOnce[F[A]] => F[A]` | `combineAllK`
+
+### Alternative
```

Comment:
Honestly, I've never put attention to this page and thus I've just realized it exists! 😊 
Would you consider mention  about `NonEmptyAlternative` there as well?

| Type | Method Name | Constraints |
| --- | --- | --- |
| `A => F[A] => F[A]` | `prependK` | |
| `F[A] => A => F[A]` | `appendK` | |

Reply:
Oh yeah sure!